spec/models/user_spec.rb

```diff
@@ -18,11 +21,30 @@
       expect(user.video_in_queue?(video)).to be(true)
     end
 
-    it 'returns fals if the video is not in users video queue' do
+    it 'returns false if the video is not in users video queue' do
       another_video = Fabricate(:video)
       expect(user.video_in_queue?(another_video)).to be(false)
     end
   end
 
+  describe '#follow' do
+    it 'creates a relationship to another user' do
+      user.follow another_user
+      expect(user.followed_users).to include(another_user)
+      expect(another_user.followers).to include(user)
+    end
+  end
 
+  describe '#unfollow' do
+
+    before(:each) do
+      user.follow another_user
+    end
```

Comment:
If there are no other specs, but 1 - i would recommend to get rid of `before` blocks. This can be easily merged to an example below. The reason is that these before blocks are performance more expensive - slowing your test suite on you have a large amount of specs.